NAMESPACE

```diff
@@ -1,18 +1,19 @@
 # Generated by roxygen2: do not edit by hand
 
 export(create_files)
-export(fill_comms_doc)
+export(get_comms_doc_info)
+export(get_debrief_doc_info)
 export(get_future_workshops)
-export(get_meta_fld)
+export(get_plan_doc_info)
```

Comment:
hey, I thought this one was gone?

Reply:
I only have get_plan_doc_info in my branch!!

Reply:
I've fixed this with the latest commit, `output_format` should have been specified in `render`